src/Hooks/Frontend.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+class Frontend implements \IWPML_Frontend_Action, \IWPML_DIC_Action {
```

Comment:
@GeorgeBotsev I don't think we need `IWPML_DIC_Action` here because there is no constructor to inject dependencies to.

Reply:
I was not sure about that.
I tested  now and it seems that you are right, so removing this now.